src/Conn.h

```diff
@@ -59,8 +59,8 @@ struct ConnTuple {
     IPAddr dst_addr;
     uint32_t src_port = 0;
     uint32_t dst_port = 0;
-    bool is_one_way = false; // if true, don't canonicalize order
     uint16_t proto = UNKNOWN_IP_PROTO;
+    bool is_one_way = false; // if true, don't canonicalize order
```

Comment:
If we wanted to play tricks, could probably also use a bit from proto and encode UNKNOWN_IP_PROTO as 2**15-1 instead. Just resolve unless this tickles you as interesting. I don't know if that's really worth it though.